src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx

```diff
@@ -18,37 +23,66 @@ const IncludeResultActions = ({
   onEdit,
   onDelete,
   onView,
+  operation,
 }: PropTypes) => {
   return (
     <Stack direction="row" alignItems="center">
       {canEdit && (
         <>
           {showDeleteAction && (
-            <Button
-              variant="outline"
-              data-testid={`delete-button-${id}`}
-              disabled={false}
-              tw="mr-4"
-              onClick={() => {
-                onDelete(id);
-              }}
-            >
-              Delete
-            </Button>
+            <>
```

Comment:
{canEdit && showDeleteAction && ()} --> I think both conditions can be simplified into one